regenerate.sh

```diff
@@ -34,43 +34,56 @@ rm -f $(find . -name '*.pb.go' | grep -v 'grpc_testing_not_regenerate')
 echo "go install google.golang.org/protobuf/cmd/protoc-gen-go"
 (cd test/tools && go install google.golang.org/protobuf/cmd/protoc-gen-go)
 
-echo "go install cmd/protoc-gen-go-grpc"
-(cd cmd/protoc-gen-go-grpc && go install .)
+ echo "go install cmd/protoc-gen-go-grpc"
+ (cd cmd/protoc-gen-go-grpc && go install .)
 
-echo "git clone https://github.com/grpc/grpc-proto"
-git clone --quiet https://github.com/grpc/grpc-proto ${WORKDIR}/grpc-proto
+ echo "git clone https://github.com/grpc/grpc-proto"
+ git clone --quiet https://github.com/grpc/grpc-proto "${WORKDIR}"/grpc-proto
 
-echo "git clone https://github.com/protocolbuffers/protobuf"
-git clone --quiet https://github.com/protocolbuffers/protobuf ${WORKDIR}/protobuf
+ echo "git clone https://github.com/protocolbuffers/protobuf"
+ git clone --quiet https://github.com/protocolbuffers/protobuf "${WORKDIR}"/protobuf
 
-# Pull in code.proto as a proto dependency
-mkdir -p ${WORKDIR}/googleapis/google/rpc
-echo "curl https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto"
-curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > ${WORKDIR}/googleapis/google/rpc/code.proto
+ # Pull in code.proto as a proto dependency
+ mkdir -p "${WORKDIR}"/googleapis/google/rpc
+ echo "curl https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto"
+ curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > "${WORKDIR}"/googleapis/google/rpc/code.proto
 
-mkdir -p ${WORKDIR}/out
+source ./protoc_installer.sh
+download_protoc
+
+export PATH="$PATH:$GOBIN"
+
+PROTOC_PATH="/usr/local/bin"
+if [[ ":$PATH:" != *":$PROTOC_PATH:"* ]]; then
+  export PATH="$PATH:$PROTOC_PATH"
+  echo "protoc added to your PATH. You might need to open a new terminal"
+else
+  echo "protoc already appears to be on your PATH"
+fi
+
+mkdir -p "${WORKDIR}"/out
 
 # Generates sources without the embed requirement
 LEGACY_SOURCES=(
-  ${WORKDIR}/grpc-proto/grpc/binlog/v1/binarylog.proto
-  ${WORKDIR}/grpc-proto/grpc/channelz/v1/channelz.proto
-  ${WORKDIR}/grpc-proto/grpc/health/v1/health.proto
-  ${WORKDIR}/grpc-proto/grpc/lb/v1/load_balancer.proto
+  "${WORKDIR}"/grpc-proto/grpc/binlog/v1/binarylog.proto
```

Comment:
```suggestion
  "${WORKDIR}/grpc-proto/grpc/binlog/v1/binarylog.proto"
```

pls, and for the other as well.